packages/nimble-components/src/anchor/specs/README.md

```diff
@@ -0,0 +1,203 @@
+
+    ```html
+    <nimble-menu>
+        <nimble-anchor-menu-item href="...">Item One</nimble-anchor-menu-item>
+        <nimble-anchor-menu-item href="...">Item Two</nimble-anchor-menu-item>
+    </nimble-menu>
+    ```
+
+-   **Tree**
+
+    The tree use case is very similar to the menu use case. I have not found examples of sites with trees (i.e. `role=tree`) with links, but based on the ARIA docs, I suspect the correct role for a link in a tree is `treeitem`. For the same reasons as the menu case, I suspect it would be cleaner and more maintainable to create a separate `nimble-anchor-tree-item`.
+
+If we are creating new components for anchors in tabs, menus, and trees, then for consistency, should we create a separate one for button-like links as well? Buttons and links are similar ARIA roles ([the](https://w3c.github.io/aria/#button) [spec](https://w3c.github.io/aria/#link) even links them to each other). We could model a button-link as just an appearance mode of the link. However, they are semantically distinct, and there are no obvious benefits to combining the two, except having one fewer component. Arguments for having a separate anchor button component include:
+
+-   consistent with existing button variations:
+    ```html
+    <nimble-button>
+        <nimble-menu-button>
+            <nimble-toggle-button>
+                <nimble-anchor-button></nimble-anchor-button></nimble-toggle-button></nimble-menu-button
+    ></nimble-button>
+    ```
+-   easily switching between standard anchor and button-like appearance (API-wise) is not a significant benefit
+-   allows us to have separate Angular directives, which gives us more freedom in case we need it
+
+For these reasons we will create a second anchor button component as part of this effort.
+
+### API
+
+[FAST's API documentation](https://github.com/microsoft/fast/blob/e576aa70c22780fffba03097277e2db9a2ec1cd8/packages/web-components/fast-foundation/src/anchor/README.md)
+
+-   _Component Name_: `nimble-anchor`
+-   _Properties/Attributes_: We will have the following properties/attributes in addition to the ones provided by the FAST anchor:
+    -   `inline`: specifies that the anchor will be used inline within a text block (as opposed to a standalone element in the UI). The anchor will always have an underline and will be formatted to align with surrounding text.
+    -   `prominent`: the "loud" version from the design doc. Colors the link text green. The name of this attribute comes from the Breadcrumb's `appearance` attribute value.
+    -   `disabled`: when set, makes the anchor inoperable and changes the styling
+-   _Methods_: Unchanged (none)
+-   _Events_: Unchanged (none)
+-   _CSS Classes and Custom Properties that affect the component_: Unchanged (none)
+-   _Slots_: We will not expose the start and end slots provided by the FAST anchor.
+
+The button-like anchor will also derive from the FAST anchor so that we use the same template and role.
```

Comment:
I may be misreading this but I'm a bit surprised that we'd use the same template for the `nimble-anchor` and `nimble-anchor-button`. Since the components look so different I assumed they'd have different templates (and that the `nimble-anchor-button` might use a lot of the same template as `nimble-button`).

If this is indeed accurate could you provide a bit more context?

Reply:
They will share CSS with the button controls, but not the template. I've added a note to help clarify that. The button's template creates a button element in the shadow root, which is not what we want for the anchor button. We need the anchor button to have an anchor element (i.e. `<a>`) in its shadow root so that it actually works like a link. That is why it shares the template with the anchor element. The differences are just styling.